configure.ac

```diff
@@ -6,7 +6,7 @@
 #
 
 # Initialization
-AC_INIT([aws-ofi-nccl], [GitHub-dev], [al-ofi-nccl-team@amazon.com], , [http://github.com/aws/aws-ofi-nccl])
+AC_INIT([aws-ofi-nccl], [1.13.0-pre], [al-ofi-nccl-team@amazon.com], , [http://github.com/aws/aws-ofi-nccl])
```

Comment:
Nack until this is on a release branch.

Reply:
How should versions be assigned to packages for development commits?

Reply:
I don't think they should be packaged, but we definitely shouldn't have a version string that's valid on master, because that has bit us in the ass in the past, and will again.

Reply:
* We cannot package it without a version.

* We want to test that packaging definitions do not break from any given change.

How should versions be assigned to packages for development commits?

Reply:
I do not want "next" to be the version in master.  If you want to set it to 999.99 or something, that's fine, I guess.

Reply:
What I would really like to do is just do our branching and tagging the way that git expects, such that `git describe` yields a reasonable version string on every commit going forward.